admin/tools/docker/build-mariadb-image.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+# Create a MariaDB+Scisql docker image.
+# Use scisql master tip
+
+# @author  Fabrice Jammes, IN2P3/SLAC
+
+set -e
+
+DIR=$(cd "$(dirname "$0")"; pwd -P)
+. "$DIR/conf.sh"
+
+MARIADB_VERSION="10.1.25"
+
+TAG="$DOCKER_REPO_MARIADB:10.1.25"
+VERSION=$(date --date='-1 month' +'%Y-%m')
+
+usage() {
+  cat << EOD
+
+  Usage: $(basename "$0") [options]
+
+  Available options:
+    -C          Rebuild the images from scratch
+    -h          This message
+
+    Create Docker images from 'qserv_latest' (default) or 'qserv-dev' eups tags.
```

Comment:
How does one chose 'qserv-dev' instead of 'qserv_latest'? Or does this description make any sense here?